libs/@blockprotocol/graph/src/codegen/parameters.ts

```diff
@@ -23,6 +23,7 @@ export type CodegenParameters = {
   typeNameOverrides?: {
     [sourceTypeId: string]: string;
   };
+  rewriteTypeId?: (typeId: VersionedUrl) => VersionedUrl;
```

Comment:
Suggest we rename this to make clear that it is only rewriting for fetching, it doesn't rewrite the id in the generated schema. I've suggested a name but anything that makes the distinction would work
```suggestion
  /**
   * Enables fetching the schema from a different URL to the one in its schema.
   * The return of this function will be used as the URL to fetch the type from.
   * Does NOT affect the id in the generated schema.
   */
  getFetchUrlFromTypeId?: (typeId: VersionedUrl) => VersionedUrl;
```

Reply:
That's a fair point, have addressed in https://github.com/blockprotocol/blockprotocol/pull/1349/commits/31ab2d8d08f7d22f51496d7ee8790b4f91e5d30a